Release_Guideline.md

```diff
@@ -0,0 +1,27 @@
+This document captures guidelines to follow when considering a new release of the BIDS specification.
+
+BIDS has generally followed a semantic versioning (semver) approach.
+The canonical semver text can be found at [https://semver.org/](https://semver.org/), and an adaptation to documents can be found at [https://semverdoc.org/](https://semver.org/).
+A specification falls somewhere between these two, so we should consider carefully the rules.
```

Comment:
"So we should carefully consider the rules" doesn't make sense to me, here. I'm really not sure what "the rules" are. Do you mean that what you're laying out below is trying to reconcile the two ?

Reply:
This was actually just a note from me to Franklin...
```suggestion
A specification falls somewhere between these two, as documentation does not require
backwards compatibility, while software has notions of bugs and features which do not
map to the reasons for updating a specification.
```